test/functional/eval/input_spec.lua

```diff
@@ -196,6 +228,19 @@ describe('input()', function()
     eq('Vim(call):E118: Too many arguments for function: input',
        exc_exec('call input("prompt> ", "default", "file", "extra")'))
   end)
+  it('supports highlighting', function()
+    feed([[:call input({"highlight": "RainBowParens"})<CR>]])
+    wait()
+    feed('(())')
+    wait()
```

Comment:
do we know why? does `input()` flush the input buffer somehow?  Real interaction will not have this API call, so it would be good to identify the issue.